src/widgets/canvas.rs

```diff
@@ -71,10 +71,6 @@ struct Layer {
 /// Braille patterns will have a resolution of 2x4 dots per cell. This means that a grid of 10x10
 /// cells will have a resolution of 20x40 dots.
 trait Grid: Debug {
-    /// Get the width of the grid in number of terminal columns
-    fn width(&self) -> u16;
-    /// Get the height of the grid in number of terminal rows
-    fn height(&self) -> u16;
```

Comment:
(Comment for rationale - no change needed) Initially this one looks a bit odd, but the trait is private, and so I'm guessing that these values are just not accessed, and so are not necessary.